src/MakerLog/Provider.php

```diff
@@ -56,8 +61,11 @@ protected function getUserByToken($token)
     protected function mapUserToObject(array $user)
     {
         return (new User())->setRaw($user)->map([
-            'id'    => $user['id'], 'nickname' => $user['username'], 'name' => $user['first_name'] . ' '. $user['last_name'],
-            'email' => null, 'avatar' => $user['avatar'],
+            'id'       => $user['id'],
+            'nickname' => $user['username'],
+            'name'     => $user['first_name'] . ' '. $user['last_name'],
+            'email'    => null,
```

Comment:
Yeah, that's a good point. It looks like I don't get the email address in the response:

![image](https://user-images.githubusercontent.com/8433587/52176360-32cf8b80-27e4-11e9-9941-1dbcecf857d5.png)

Reply:
Hello @matteing, 

is it intended to not provide the email address? Just asking as I've got it registered with makerlog.

-- Peter

Reply:
You should probably ask on the API chat

Reply:
Hey @m1guelpf 

have you been able to find out more about the email address? As far as I know the specs of the API there isn't a way to receive the email address. 

Also, do you know if we need to refresh the token regularly? If so, at which endpoint?

-- Peter

Reply:
I have only played a bit with the API (the only reason I know there's a way to get email is because I pestered Sergio for days until he added it), so please ask this to Sergio directly.

Reply:
Okay, I'll ask him. Sergio seems fairly busy atm and I'm not sure how quick we get an answer.  Should we merge this and open a new PR, if we get an option?

Reply:
Okay, I've had a chat with Sergio. We need a second call to retrieve the email address. Do you know a provider which requires a second call for anything? Just want to make sure it's as close as possible to the existing code base.

Reply:
I'm not familiar with _all_ the providers, but I don't think so. It'd still be nice to get the email tho

Reply:
Hello @m1guelpf,

I've added the email request similar to a different provider and cleaned the code a bit up. Let me know if you think we need to do anything more.

Thanks,
Peter